networkx/drawing/layout.py

```diff
@@ -1121,6 +1122,229 @@ def multipartite_layout(G, subset_key="subset", align="vertical", scale=1, cente
     return pos
 
 
+def forceatlas2_layout(
+    G,
+    pos=None,
```

Comment:
Since this has so many parameters that can be tweaked, I think it's worth enforcing keyword-only arguments here. I'd vote for everything except `G` to be keyword-only, though I'm not super picky here - for example I can see the argument for putting the `*` after `pos` instead of before.

```suggestion
    G,
    *,
    pos=None,
```

Reply:
Just for clarification, do you mean here to replace the many input parameters to a ``**kwargs`` and only document the possible parameters?

Reply:
No... the intent is to use the "new" python feature of forcing users to use the keyword name when they specify an argument. So a function
```python
def func(p1, p2, *, kw1, kw2):
```
can only be called as `func(3, 4, kw1=6)`, not as `func(3, 4, 6)`.  This leads to more readable code -- the reader doesn't have to count out which argument is which. The name says which is which.  keyword-only means it has to be named in the function call. position-or-keyword means if the name isn't provided it can work based on the position in the function call.

We are trying to move to keyword-only function signatures in NetworkX functions that have a lot of parameters.  

I think @rossbar would have just added this change, but wanted your opinion about whether the `pos` input should be a position-or-keyword parameter or a keyword only.
`*_layout(G, pos)` vs `*_layout(G, pos=pos)`  (imagine lots of other parameters provided too).  

My intuition is weak on this question. I guess I would prefer to leave pos as position-or-keyword and put the `*` symbol after `pos` in the function signature. But it really is up to you. We'd like the `*` to be added somewhere in there though. :)

Reply:
Indeed, @dschult is correct about my intent - sorry for the confusion. I went ahead and pushed up the suggestion incorporating @dschult 's feedback so I'll go ahead and mark this as "resolved"

Reply:
Thanks for the clarification.

Reply:
I was not aware of this way of parsing keywords (other than the * and **), nice to have in my toolkit!